dvc/repo/__init__.py

```diff
@@ -202,6 +202,18 @@ def __init__(
     def __str__(self):
         return self.url or self.root_dir
 
+    @staticmethod
+    @contextmanager
+    def open(url, *args, **kwargs):
+        if os.path.isdir(url):
```

Comment:
just an `exists` check would be enough here, I think.

Reply:
Good point, might improve the local error.